docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,303 @@
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root c731613c-7a61-11e4-8b8c-42010a863922
+ 0:00:01    R    67.79 MB  0.25  105.93   none  root c7310460-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    67.62 MB  0.5   105.66   none  root bda74154-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root bda6e97f-7a61-11e4-8b8c-42010a863922
+ 0:00:03    R    68.20 MB  0.75  106.57   none  root b0e206f1-7a61-11e4-8b8c-42010a863922
+```
+The number of Kubernetes pods listed earlier (from `bin/kubectl get pods`) should equal to the number active Mesos tasks listed the previous listing (`mesos ps`).
+
+Next, determine the internal IP address of the front end [service portal][8]:
+
+```bash
+$ bin/kubectl get services
+NAME            LABELS                                    SELECTOR            IP             PORT
+kubernetes      component=apiserver,provider=kubernetes   <none>              10.10.10.2     443
+kubernetes-ro   component=apiserver,provider=kubernetes   <none>              10.10.10.1     80
+redismaster     <none>                                    name=redis-master   10.10.10.49    10000
+redisslave      name=redisslave                           name=redisslave     10.10.10.109   10001
+frontend        <none>                                    name=frontend       10.10.10.149   9998
+```
+
+Interact with the frontend application via curl using the front-end service IP address from above:
+
+```bash
+$ curl http://${frontend_service_ip_address}:9998/index.php?cmd=get\&key=messages
+{"data": ""}
+```
+
+Or via the Redis CLI:
+
+```bash
+$ sudo apt-get install redis-tools
+$ redis-cli -h ${redis_master_service_ip_address} -p 10000
+10.233.254.108:10000> dump messages
+"\x00\x06,world\x06\x00\xc9\x82\x8eHj\xe5\xd1\x12"
+```
+#### Test Guestbook App
+Or interact with the frontend application via your browser, in 2 steps:
+
+First, open the firewall on the master machine.
+
+```bash
+$ # determine the internal port for the frontend service portal
+$ sudo iptables-save|grep -e frontend  # -- port 36336 in this case
+-A KUBE-PORTALS-CONTAINER -d 10.10.10.149/32 -p tcp -m comment --comment frontend -m tcp --dport 9998 -j DNAT --to-destination 10.22.183.23:36336
+-A KUBE-PORTALS-CONTAINER -d 10.22.183.23/32 -p tcp -m comment --comment frontend -m tcp --dport 9998 -j DNAT --to-destination 10.22.183.23:36336
+-A KUBE-PORTALS-HOST -d 10.10.10.149/32 -p tcp -m comment --comment frontend -m tcp --dport 9998 -j DNAT --to-destination 10.22.183.23:36336
+-A KUBE-PORTALS-HOST -d 10.22.183.23/32 -p tcp -m comment --comment frontend -m tcp --dport 9998 -j DNAT --to-destination 10.22.183.23:36336
+
+$ # open up access to the internal port for the frontend service portal
+$ sudo iptables -A INPUT -i eth0 -p tcp -m state --state NEW,ESTABLISHED -m tcp \
+  --dport ${internal_frontend_service_port} -j ACCEPT
+```
+
+Next, add a firewall rule in Google Cloud Platform Console / Networking:
```

Comment:
I would be a bit more specific, maybe say something like "choose Compute > Compute Engine > Networks, then click on the name of your mesosphere-* network, then click 'New firewall rule'"